Modules/timemodule.c

```diff
@@ -105,7 +105,7 @@ _PyFloat_FromPyTime(PyTime_t t)
 
 
 static PyObject *
-time_time(PyObject *self, PyObject *unused)
+time_time(PyObject *self, PyObject *Py_UNUSED(unused))
```

Comment:
I'd suggest renaming `unused` to `args` since the method is marked as a `METH_NOARGS`, `METH_O` or `METH_VARARGS`. And the signature of such method is usually 

```C
static PyObject *
method(PyObject *self, PyObject *args);
```

So I suggest having something:

```C
static PyObject *
method(PyObject *self, PyObject *Py_UNUSED(args));
```

Reply:
That's good idea. I'd love to do that, but I kind of don't want to rename it (I just want to be consistent with the original argument name)

Reply:
Yes but it's better to be consistent with the CPython project and convene the intent of the function a method (there is hardly any `Py_UNUSED(unused)` occurrences across the project but there are a lot of `Py_UNUSED(args)` occurrences).

Reply:
You're right, I have made changes.